app/controllers/issues/socializations_controller.rb

```diff
@@ -0,0 +1,17 @@
+module Issues
+  class SocializationsController < ApplicationController
+    def follow
+      @issue = Issue.find(params[:issue_id])
+      current_user.follow!(@issue)
+      @issue.reload
```

Comment:
but you are redirecting to @issue page. does't it will be updated on page opening?